README.md

```diff
@@ -1 +1,91 @@
-# jp2_offline
\ No newline at end of file
+# jp2_offline
+
+This application was build together with [Juan Pablo II online](https://github.com/erikiado/jp2_online).
+
+# Table of contents
+
+1. [System Requirements](#system)
+2. [For Developers](#developers)
+  1. [Quick Start](#quick)
+  2. [Build the application](#build)
+  3. [Test](#test)
+3. [For users](#user)
+
+# System Requirements
+
+# For Developers
+
+## Quick start
+
+```shell
+  # Clone the Quick Start repository
+  $ git clone https://github.com/Ral15/jp2_offline/
+
+  # Go into the express app
+  $ cd jp2_offline/express-app
+
+  # Install the dependencies and run
+  $ npm install
+
+  # Return to the repository
+  $ cd ..
+
+  # Install the dependencies and run
+  $ npm install && npm start
+```
+
+## Build the application
+
+To Build the application we used Electron Packager.
+To create the executable just do:
+
+```shell
+  npm build
+```
+
+This creates a executable for windows x64. To changes architecture and operating system you can run
+
+```shell
+  # To create the same executable
+  electron-packager . "San Juan Pablo II" --platform=win32 --arch=x64
```

Comment:
Yes, but you need to install Wine. That's why I put the link to Electron packager for more information